lib/util/util.c

```diff
@@ -1504,7 +1507,10 @@ bool util_sscanf_double(const char * buffer , double * value) {
    Base 8
 */
 
-bool util_sscanf_octal_int(const char * buffer , unsigned int * value) {
+bool util_sscanf_octal_int(const char * buffer , int * value) {
```

Comment:
@joakim-hove I am not completely sure about this change. I think using `int` is the correct solution (for consistency with other methods, and I see no reason why octal should not be negative). On the other hand, I found at least one place in libres where this is used with a variable of type mode_t (which is unsigned - this leads only to a warning, things seems to work fine). 

Any suggestion? Maybe we could add a second version `util_sscanf_octal_uint` that handles unsigned int